examples/makeSolver.py

```diff
@@ -12,6 +12,7 @@ def parse_setup(setup_file: str):
 
 def main(robot_type, setup_file):
     setup = parse_setup(setup_file)
+    setup['robot']['urdf_file'] = current_path = os.path.dirname(os.path.abspath(__file__)) + "/assets/"+str(robot_type)+"/" + setup['robot']['urdf_file']
```

Comment:
what is the variable `current_path` being used for?

Reply:
the variable `current_path` is not used and can be removed.